public/javascripts/SVLabel/src/SVLabel/data/Form.js

```diff
@@ -276,24 +278,19 @@ function Form (labelContainer, missionModel, missionContainer, navigationModel,
             asyncParam = false;
 
         // Old code: this does not work on the newest versions of Google Chrome.
```

Comment:
You can remove this line. Also remove the `asyncParam` above along with that big paragraph I wrote.